app/database/models/recipesModel.js

```diff
@@ -0,0 +1,35 @@
+const db = require('../dbConfig');
+
+/**
+ * insert recipe as a creator
+ * @param recipe
+ * @param uid
+ * @returns {Promise<{insertStatus: string}>}
+ */
+async function insertAsCreator(recipe, uid){
+    const result = {insertStatus: 'failed'};
+    await db.transaction(async (transaction) => {
+        try {
+            const ids = await db('recipes')
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 6 spaces but found 12.](https://app.codacy.com/gh/cse110-fa21-group17/cse110-fa21-group17/pullRequest?prid=8445077)
```suggestion
      const ids = await db('recipes')
```